Parser/action_helpers.c

```diff
@@ -1071,6 +1071,19 @@ _PyPegen_check_legacy_stmt(Parser *p, expr_ty name) {
     return 0;
 }
 
+int
+_PyPegen_check_fstring_conversion(Parser *p, Token* symbol, expr_ty conv) {
+    if (symbol->lineno != conv->lineno || symbol->end_col_offset != conv->col_offset) {
```

Comment:
Can you post an example on how to trigger this?

Reply:
For the line check:
```py
print(f"""{
        1
        !
        s
}""")
```

For the column check:
```py
print(f"{1! s}")
```

Reply:
Could you add them as tests or are we already covering them?

Reply:
We already cover them (this is one of the test failures)